src/components/SnapshotDetailView.jsx

```diff
@@ -89,8 +97,18 @@ export class SnapshotDetailView extends React.PureComponent {
     dispatch(removeCustodianFromSnapshot(snapshot.id, removeableEmail));
   };
 
+  handleFilterDatasets = (limit, offset, sort, sortDirection, searchString) => {
```

Comment:
Note: `handleFilterDatasets` for `SnapshotDetailView` is different from that in `DatasetView` because each snapshot has its own list of datasets as part of the snapshot object, so filtering and sorting are just manipulations on that array.